components/admin_console/plugin_management/plugin_management.test.jsx

```diff
@@ -15,6 +15,8 @@ describe('components/PluginManagement', () => {
                 Enable: true,
                 EnableUploads: true,
                 AllowInsecureDownloadUrl: false,
+                EnableMarketplace: true,
+                MarketplaceURL: 'www.marketplace.com',
```

Comment:
nit: suggest using `marketplace.example.com` for any tests. In case the code ever happens to resolve, it's best not to potentially spam a "real" domain.